models/Product.js

```diff
@@ -10,6 +10,7 @@ const ProductSchema = new mongoose.Schema(
     price: {
       type: Number,
       required: [true, 'Please provide product price'],
+      // might be good to have min price validation
```

Comment:
Just a suggestion to also have a `min` validation here for this field, since you likely don't want to allow saving negative prices.

Reply:
@akosasante I put it in almost all fields (min:0), is that ok?

Reply:
Yep that seems reasonable!